clients/src/main/java/org/apache/kafka/clients/consumer/internals/RequestState.java

```diff
@@ -98,12 +93,11 @@ public boolean canSendRequest(final long currentTimeMs) {
      * is a request in-flight.
      */
     public boolean requestInFlight() {
-        return this.lastSentMs > -1 && this.lastReceivedMs < this.lastSentMs;
+        return requestInFlight;
```

Comment:
do you think it would make sense to rename this to: `inflightRequested`? `requestInflight` sounds like a verb/action to me.

Reply:
`inflightRequested` seems to change the meaning. Who requested an inflight?

In my head it's clearly the noun "request", but if you are bothered, may I suggest "isRequestInFlight"?

Reply:
@philipnee—I didn't make any changes to the name as `requestInFlight` was the existing method name. Are you OK to leave this for now?

Reply:
@philipnee—let us know if you're OK to leave the method name as is. Thanks!